atom_batch_execution/scripts/batch_execution

```diff
@@ -73,8 +82,15 @@ def main():
         file_content = f.read()
         data = yaml.safe_load(file_content)
 
-    # Load dataset
-    dataset = loadJSONFile(data['dataset_path'] + '/dataset.json')
+    dataset = loadJSONFile(data['dataset_path'])
+
+    collections_to_remove = data['collections_to_remove']
+
+    args["collection_selection_function"] = lambda x : int(x) not in collections_to_remove
```

Comment:
Is this really needed? I think we are creating two different ways to do the same thing. I know this is a wrapper on the old way, but I suggest we are consistent in the code base and all of our scripts should function in the same way to select collections.

Reply:
Initially when I started working with the batch executions, I questioned myself the same thing. When I first worked on batch executions, this line of code was already in the `data.yml` template. (Line 19)


https://github.com/lardemua/atom/blob/16275110d396ca32aa2175081593b341e47b8183/atom_batch_execution/experiments/softbot_example/data.yml#L17-L19

 It was not functional however. I can recall discussing this with @miguelriemoliveira and we ended up deciding on leaving it there and make it functional to allow greater flexibility. 


The user could instead use the `remove_collections_from_atom_dataset` instead, but it wouldn't be on the fly so it could in some cases be cumbersome. Also I see this script more geared towards removing collections that are purely bad and never going to be used.

Picture one use case where the user wanted to run 2 experiments where he used half of the dataset in each experiment, to test for example the impact of odometry error or something similar. With this on the fly functionality its a easy thing to do. Also other scripts such as calibrate or the evaluations allow to select collections on the fly, so I think the same logic applies here.

It also filters the collections in the same standardized way, using :

https://github.com/lardemua/atom/blob/fc1ea8fa0aad4b0495d968753a43f594d1604064/atom_batch_execution/scripts/batch_execution#L93